containers/sfdx-project/README.md

```diff
@@ -0,0 +1,53 @@
+# SFDX Project
```

Comment:
Here's where i need the most feedback...documentation.  

Some specific comments/notices:
1. Contributors
Salesforce Developer Experience Team ( i wanted to include PDT, PWT ) but wasn't sure exactly what to put here...

2. Link to the official salesforce image.
Once the official image gets published, I'll replace the general docker hub registry link, or maybe this should be a link to the public repo?

3. I also need to link to the official documentation page for sfdx remote dev that bindu is working on, which isn't up yet.  I think it's ok to push this up for now with a link to the general documentation, but we can submit an updated PR with the final "deep link" to remote development documentation once bindu is finished.

4. Lastly, there is a microsoft copyright ( MIT ) on here that might need legal review?

Reply:
Add Bindu to this?

Reply:
Yeah, i invited her as a collaborator to review this.

Reply:
As for 4, yes this is fine.